src/Microsoft.AspNet.Http/HttpContextFactory.cs

```diff
@@ -9,6 +9,10 @@ public class HttpContextFactory : IHttpContextFactory
     {
         private IHttpContextAccessor _httpContextAccessor;
 
+        public HttpContextFactory() : this(null)
```

Comment:
This was addressed as part of https://github.com/aspnet/DependencyInjection/issues/69